src/main/java/com/autotune/analyzer/workerimpl/BulkJobManager.java

```diff
@@ -162,14 +168,38 @@ public void run() {
                     notification.setMessage(String.format(notification.getMessage(), e.getMessage()));
                     setFinalJobStatus(FAILED, String.valueOf(HttpURLConnection.HTTP_BAD_REQUEST), notification, datasource);
                 }
+
+                //check for "is_ros_enabled" flag and specify
+                if ((is_ros_enabled) && (null == this.bulkInput.getMetadata_profile())) {
+                    metadataProfile = MetadataProfileCollection.getInstance().
+                            loadMetadataProfileFromCollection(AnalyzerConstants.MetadataProfileConstants.CLUSTER_METADATA_LOCAL_MON_PROFILE);
```

Comment:
Please make use 
 CreateExperimentConfigBean  to set defaults 
 
 example
 loadMetadataProfileFromCollection(CREATE_EXPERIMENT_CONFIG_BEAN.getDefaultMetaDataProfile)